src/openjd/adaptor_runtime/_entrypoint.py

```diff
@@ -64,7 +88,6 @@
     os.path.join(
         _system_config_path_prefix,
         "openjd",
-        "worker",
```

Comment:
This is also a breaking change right? Just want to make sure we call it out in the commit so that it is available in the changelog

Reply:
Right, I'll update the commit